### PR TITLE
[nnyeah] Arithmetic tests

### PR DESCRIPTION
**Pull request by stephen-hawley**
Added tests for arithmetic and basic casting

### CODE REVIEW

File: tools/nnyeah/integration/API/IntegrationAPI.cs [hunk 1]
```diff
@@ -36,6 +36,10 @@ public NIntAPI ()
 #pragma warning disable CS0067 // The event 'NIntAPI.Event' is never used
         public event EventWithNInt Event;
 #pragma warning disable CS0067 // The event 'NIntAPI.Event' is never used
+	public nint Sum (nint a, nint b) => a + b;
+	public nint Prod (nint a, nint b) => a * b;
+	public long ToLong(nint a) => (long)a;
```
Comment (tj-devel709) [inline tools/nnyeah/integration/API/IntegrationAPI.cs:41]:
```suggestion
	public long ToLong (nint a) => (long)a;
```

File: tools/nnyeah/integration/API/IntegrationAPI.cs [hunk 2]
```diff
@@ -50,6 +54,10 @@ public NUIntAPI ()
 #pragma warning disable CS0067 // The event 'NUIntAPI.Event' is never used
         public event EventWithNUInt Event;
 #pragma warning disable CS0067 // The event 'NUIntAPI.Event' is never used
+	public nuint Sum (nuint a, nuint b) => a + b;
+	public nuint Prod (nuint a, nuint b) => a * b;
+	public long ToLong(nuint a) => (long)a;
```
Comment (tj-devel709) [inline tools/nnyeah/integration/API/IntegrationAPI.cs:59]:
```suggestion
	public long ToLong (nuint a) => (long)a;
```

File: tools/nnyeah/integration/API/IntegrationAPI.cs [hunk 3]
```diff
@@ -64,5 +72,8 @@ public NFloatAPI ()
 #pragma warning disable CS0067 // The event 'NFloatAPI.Event' is never used
         public event EventWithNFloat Event;
 #pragma warning disable CS0067 // The event 'NFloatAPI.Event' is never used
+	public nfloat Sum (nfloat a, nfloat b) => a + b;
+	public nfloat Prod (nfloat a, nfloat b) => a * b;
+	public double ToDouble(nfloat a) => (double)a;
```
Comment (tj-devel709) [inline tools/nnyeah/integration/API/IntegrationAPI.cs:77]:
```suggestion
	public double ToDouble (nfloat a) => (double)a;
```

**Review by tj-devel709 — COMMENTED** [review]
A few style suggestions, but everything else LGTM!

**Review by rolfbjarne — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]
Approve but I would double down on using rebase instead of merge to ease the commit by commit review.

**Pull request merged**